unit_tests/ut_h_type_hints.amlg

```diff
@@ -0,0 +1,28 @@
+(seq
+	#unit_test (direct_assign_to_entities (assoc unit_test (load "unit_test.amlg")))
+	(call (load "unit_test_howso.amlg") (assoc name "ut_h_type_hints.amlg"))
+
+    ;explicitly allow access to internal method for this test
+	(accum_entity_roots "howso" (list
+		(set_labels
+			(lambda
+				(call !BuildParameterValidationMap)
+			)
+			(list "BuildParameterValidationMap")
+		)
+	))
+
+    (declare (assoc
+        invalid_type_hints (call_entity "howso" "BuildParameterValidationMap")
+    ))
+
+    (call assert_true (assoc
+        obs (= (true) invalid_type_hints)
```

Comment:
you don't need the (= (true)) here because that's what the assert_true tests, just do `obs invalid_type_hints`

Reply:
well.. the behavior I need to capture here is if `invalid_type_hints` is (true) or if it is a (list) of stuff.  Both give a PASS on `#assert_true`. Hence why I have the specific `(= (true) _)` here. I'll keep it this way for now, but it may be worth tightening the `#assert_true` logic to specifically check if the value is `(true)` rather than if it is a "truthy".